src/devices/Bmx280/Bme280.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Device.I2c;
+
+namespace Iot.Device.Bmx280
+{
+    public class Bme280 : BmxBase
+    {
+        private const byte Signature = 0x60;
```

Comment:
Similar to the Address comment, should this be something more explicit like `DeviceID` or somehting like that?

Reply:
Yes, I can rename this to ```DeviceId```